src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -390,13 +416,53 @@ private static MappedTranscriptIdInfo createMappedTranscriptIdInfo( final SAMSeq
             }
         }
 
+        // TODO: This may not be correct.  It's not clear that the whole sequence should be taken.  Perhaps it should be handled with logic downstream...
```

Comment:
Has this TODO been addressed?

Reply:
Not yet.  I haven't thought through what should be done if there is no CDS.

In practice I'm not sure that this case actually comes up.